config/secrets.yml

```diff
@@ -12,6 +12,8 @@
 
 development:
```

Comment:
This file should not be in source control.

Literally ALL your secret keys are now public.